tests/conftest.py

```diff
@@ -1,6 +1,12 @@
 import httpretty
 import pytest
 from h_matchers import Any
+from pytest_factoryboy import register
+
+from tests.factories import TranscriptFactory
+
+# Each factory has to be registered with pytest_factoryboy.
+register(TranscriptFactory)
```

Comment:
No way around this AFAICT. `pytest_factoryboy` comes with an `@register` decorator that you can just put on the factory classes where you define them, but it doesn't seem to work.